test/spec/ol/format/kmlformat.test.js

```diff
@@ -1309,8 +1309,80 @@ describe('ol.format.KML', function() {
         expect(f.get('capital')).to.be('London');
         expect(f.get('population')).to.be('60000000');
       });
+
+      it('can read ExtendedData with displayName instead of name', function() {
+        var text =
+            '<kml xmlns="http://earth.google.com/kml/2.2">' +
+            '  <Placemark xmlns="http://earth.google.com/kml/2.2">' +
+            '    <ExtendedData>' +
+            '      <Data>' +
+            '        <displayName>capital</displayName>' +
+            '        <value>London</value>' +
+            '      /Data>' +
+            '    </ExtendedData>' +
+            '  </Placemark>' +
+            '</kml>';
+        var fs = format.readFeatures(text);
+        expect(fs).to.have.length(1);
+        var f = fs[0];
+        expect(f).to.be.an(ol.Feature);
+        expect(f.get('capital')).to.be('London');
+      });
+
     });
 
+    describe('region ', function() {
+      it('can read LatLonAltBox', function() {
+        var text =
+          '<kml xmlns="http://www.opengis.net/kml/2.2">' +
+          '  <Document>' +
+          '  <Placemark>' +
+          '    <Region>' +
+          '      <LatLonAltBox>' +
+          '        <north>43.507038</north>' +
+          '        <south>43.488125</south>' +
+          '        <east>1.326859</east>' +
+          '        <west>1.302759</west>' +
+          '      </LatLonAltBox>' +
+          '    </Region>' +
+          '  </Placemark>' +
+          '  </Document>' +
+          '</kml>';
+        var fs = format.readFeatures(text);
+        expect(fs).to.have.length(1);
+        var f = fs[0];
+        expect(f).to.be.an(ol.Feature);
+        expect(f.get('north')).to.be(43.507038);
```

Comment:
Would it maybe make make sense to create an [ol.Extent](http://openlayers.org/en/master/apidoc/ol.html#.Extent) instead of creating a property with each value?